go-controller/pkg/config/config.go

```diff
@@ -1246,6 +1247,14 @@ var OvnKubeNodeFlags = []cli.Flag{
 		Value:       OvnKubeNode.MgmtPortNetdev,
 		Destination: &cliConfig.OvnKubeNode.MgmtPortNetdev,
 	},
+	&cli.StringFlag{
```

Comment:
Can we infer the representor name when netdev name is known/provided?
It seems that there will only be one scenario rep name is used:
1) ovnkube-node in full mode
2) netdev name is provided

so we shall be able to know when to infer the rep device.

Reply:
Yes, it is possible. I'll try to add it.

Reply:
So will the number of required parameters decrease from
```
      OVNKUBE_NODE_MGMT_PORT_NETDEV:       ens1f0v0
      OVNKUBE_NODE_MGMT_PORT_REPRESENTOR:  ens1f0_0
```
to:
```
      OVNKUBE_NODE_MGMT_PORT_NETDEV:       ens1f0v0
```
OVNKUBE_NODE_MGMT_PORT_REPRESENTOR will be automatically determined if the netdev is a VF?

Reply:
Yes. But it will be determined only if node mode is full and only for certain devices - VFs and SFs in future patchset.